dissect/target/plugins/os/unix/bsd/citrix/_os.py

```diff
@@ -117,10 +138,19 @@ def users(self) -> Iterator[UnixUserRecord]:
                 # the 'root' user as having '/root' as a home, not in /var/nstmp.
                 user_home = "/root"
 
+            seen.add(username)
             yield UnixUserRecord(name=username, home=user_home)
 
         for username in nstmp_users:
-            yield UnixUserRecord(name=username, home=nstmp_user_path.format(username=username))
+            seen.add(username)
+            yield UnixUserRecord(name=username, home=nstmp_path.joinpath(username))
+
+        # Yield users from /etc/shadow if we have not seem them in previous loops
+        for user in super().users():
+            if user.name in seen:
+                continue
+            user_home = user.home if user.home != "/" else None
+            yield UnixUserRecord(name=user.name, home=user_home)
```

Comment:
You lose all the other fields from the original record by doing this (`gecos`, `shell`, etc).